src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Extensions/DesignTimeDirectiveTargetExtension.cs

```diff
@@ -158,7 +158,7 @@ private void WriteDesignTimeDirectiveToken(CodeRenderingContext context, DesignT
                     var stringSyntax = context.DocumentKind switch
                     {
                         "mvc.1.0.razor-page" => "Route",
-                        "component.1.0" => "ComponentRoute",
+                        "component.1.0" => "Route,Component",
```

Comment:
Is the ASP.NET side of this merged and looking for both tags now? or do we need to coordinate things?

Reply:
The change is so both syntaxes use the "Route" tag, and component routes have a component argument.

The ASP.NET Core change is in progress. They don't need to perfectly sync up.